internal/pkg/manifest/workload.go

```diff
@@ -400,6 +400,7 @@ type PublishConfig struct {
 // Topic represents the configurable options for setting up a SNS Topic.
 type Topic struct {
 	Name *string `yaml:"name"`
+	Type *string `yaml:"type"`
```

Comment:
What do you think of always setting a default value for `Type` here and removing the need of leaking "standard" to `stack/transformers.go`.

For example: https://go.dev/play/p/ADtA1hKgF7s

Reply:
That's a great idea. I like it. I was contemplating where I should set the default value.